### DISCUSSION TITLE
No song information shown in Web Player when playing in random mode

### DISCUSSION THREAD

**Discussion by jgdye** [ampache/ampache#4071] [Q&A]
I'm posting in discussion because I'm not sure if this is operating as intended or not.  When selecting "Random Play" from a playlist, the player doesn't show any queue and lists the currently playing song as "Random" by "Ampache."  I can understand the part about no queue as Ampache may not know what it's playing next (depending on how the random play is implemented), but is it intentional that there is no currently playing information?  The "Now Playing" and "Recently Played" sections on the home page work as expected, so why not the player at the bottom of the screen? I'm using Ampache 7.0.0 on Debian.

![player](https://github.com/user-attachments/assets/2e24d377-81b2-403e-8a65-33fe9d2a1b96)

**Comment by lachlan-00** [comment]
it's intended because i don't know how to do it properly :|

random links direct to the play url so they should be able to look up the data but i haven't looked hard enough into jplayer / JS to work out how to do it

**Reply by lachlan-00** [reply]
now one reason why everything like ratings and art don't show up AFTER you play a normal song is the random player doesn't load the areas for the regular jplayer items,

i might fix that though cause i can probably just put empty data in there
![image](https://github.com/user-attachments/assets/bfb3cb7b-95b7-455b-9c13-43b9d29c6188)
